app/processing/payment-request/get-completed-payment-request-by-payment-request-id.js

```diff
@@ -0,0 +1,22 @@
+const db = require('../../data')
+const { COMPLETED } = require('../../constants/statuses')
+
+const getCompletedPaymentRequestByPaymentRequestId = async (paymentRequestId, transaction) => {
+  return db.paymentRequest.findOne({
+    transaction,
+    lock: true,
```

Comment:
I don't think we need to lock record for read purpose.

Reply:
Correct -- my naive copy-pasta approach